lis/surfacemodels/land/noahmp.3.6/da_soilm_sig0VV/noahmp36_qc_Sig0VVobs.F90

```diff
@@ -216,6 +219,11 @@ subroutine noahmp36_qc_Sig0VVobs(n,k,OBS_State)
        vegt,&
        vegt_obs)
 
+  call LIS_convertPatchSpaceToObsSpace(n,k,&
```

Comment:
Here you define new variables (_slope_ and _slope_obs_) and you call this routine (LIS_convertPatchSpaceToObsSpace). Why would it only be needed for VV and not for the other observations (VH, VVVH...)?

Reply:
Great catch. It's needed for all, I only tested the vv case and made a mistake by implementing it for the others.